Demo/src/main/res/layout/pay_later_button_test_layout.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- XML view for testing only. These aren't used anywhere else in the demo application. -->
```

Comment:
Should we add another page in the demo app showing all the different color variations? I'm not sure if our documentation has all of the variants. It might make it easier for merchants to decide on which color they should use in their app.